tools/integration-test/src/tests/client_settings.rs

```diff
@@ -0,0 +1,125 @@
+use std::time::Duration;
+
+use ibc::core::ics02_client::trust_threshold::TrustThreshold;
+
+use ibc::clients::ics07_tendermint::client_state::ClientState as TendermintClientState;
+use ibc::core::ics02_client::client_state::AnyClientState;
+use ibc::Height;
+use ibc_relayer::chain::client::ClientSettings;
+use ibc_relayer::chain::cosmos;
+
+use ibc_test_framework::prelude::*;
+
+/// A test to exercise default foreign client settings.
+#[test]
+fn test_client_defaults() -> Result<(), Error> {
+    run_binary_chain_test(&ClientDefaultsTest)
+}
+
+/// A test to exercise customization of foreign client settings.
+#[test]
+fn test_client_settings() -> Result<(), Error> {
+    run_binary_chain_test(&ClientSettingsTest)
+}
+
+struct ClientDefaultsTest;
+
+struct ClientSettingsTest;
+
+struct SettingsTestOverrides;
+
+impl TestOverrides for ClientDefaultsTest {
+    fn modify_relayer_config(&self, config: &mut Config) {
```

Comment:
What is the purpose of this config override? When I comment out this override, the test still passes.

Reply:
Really? I get an assertion failure as expected:

```
Message:  assertion failed: `(left == right)`
  left: `10s`,
 right: `24s`
Location: tools/integration-test/src/tests/client_settings.rs:55
```

Reply:
Ah I missed the fact that there are actually two tests in the file. Perhaps we should consider whether to group multiple independent tests in the same file inside nested `mod {}` for easier identification.